frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/DeleteBranchOnSlideOutSuggestionDialog.kt

```diff
@@ -31,61 +31,60 @@ class DeleteBranchOnSlideOutSuggestionDialog(project: Project) :
 
   override fun createActions(): Array<Action?> = emptyArray()
 
-  override fun createCenterPanel() =
```

Comment:
what happened here?

Reply:
It looks that **spotless** likes to change Kotlin style with every minor release... that's not the first time I see random changes like that